bip-musig2.mediawiki

```diff
@@ -67,7 +67,7 @@ When implementing the specification, make sure to understand this section thorou
 
 The basic order of operations to create a multi-signature with the specification is as follows:
 The signers start by exchanging public keys and computing an aggregate public key using the ''KeyAgg'' algorithm.
-When they want to sign a message, each signer starts the signing session by running ''NonceGen'' to compute ''secnonce'' and ''pubnonce''.
+When they want to sign a message, each signer starts the signing session by running ''NonceGen'' to compute ''secnonce'' and ''pubnonce''<ref>We treat the 64-byte ''secnonce'' and 66-byte ''pubnonce'' as grammatically singular despite the fact that they are serializations of two scalars and two elliptic curve points respectively. This treatment may be confusing for readers familiar with with MuSig2 paper. However, serialization is a technical detail that is irrelevant for users of MuSig2 interfaces.</ref>.
```

Comment:
micro nit:
```suggestion
When they want to sign a message, each signer starts the signing session by running ''NonceGen'' to compute ''secnonce'' and ''pubnonce''<ref>We treat the 64-byte ''secnonce'' and 66-byte ''pubnonce'' as grammatically singular despite the fact that they are serializations of two scalars and two elliptic curve points, respectively. This treatment may be confusing for readers familiar with with MuSig2 paper. However, serialization is a technical detail that is irrelevant for users of MuSig2 interfaces.</ref>.
```

Reply:
done

Reply:
another nit: replace "with with MuSig2 paper" with "with the MuSig2 paper"